qa/standalone/scrub/osd-scrub-test.sh

```diff
@@ -576,12 +576,12 @@ function TEST_dump_scrub_schedule() {
 
     sleep 5
     sched_data=()
-    declare -A expct_qry_duration=( ['query_last_duration']="0" ['query_last_duration_neg']="not0" )
-    wait_any_cond $pgid 10 $saved_last_stamp expct_qry_duration "WaitingAfterScrub " sched_data || return 1
+    declare -A expect_qry_duration=( ['query_last_duration']="0" ['query_last_duration_neg']="not0" )
+    wait_any_cond $pgid 10 $saved_last_stamp expect_qry_duration "WaitingAfterScrub " sched_data || return 1
     # verify that 'pg dump' also shows the change in last_scrub_duration
     sched_data=()
-    declare -A expct_dmp_duration=( ['dmp_last_duration']="0" ['dmp_last_duration_neg']="not0" )
-    wait_any_cond $pgid 10 $saved_last_stamp expct_dmp_duration "WaitingAfterScrub_dmp " sched_data || return 1
+    declare -A expect_dmp_duration=( ['dmp_last_duration']="0" ['dmp_last_duration_neg']="not0" )
+    wait_any_cond $pgid 10 $saved_last_stamp expect_dmp_duration "WaitingAfterScrub_dmp " sched_data || return 1
```

Comment:
Leave this alone, shortening in code is not uncommon, we gain nothing by changing it.